.gitignore

```diff
@@ -97,4 +97,5 @@ typings/
 
 
 # End of https://www.gitignore.io/api/node
-.env 
+.env
+routes.http
```

Comment:
Why is this ignored

Reply:
This file is for testing using REST Client in VSCode that's why I have ignored it.